src/public/media.ts

```diff
@@ -22,6 +22,7 @@ import {
   scanBarCodeAPIMobileSupportVersion,
   nonFullScreenVideoModeAPISupportVersion,
 } from '../internal/constants';
+import { userOriginUrlValidationRegExp } from '@microsoft/teams-js';
```

Comment:
Why are you importing this but then not using it anywhere?

Reply:
@ydogandjiev This wasn't intentional, Will remove this import..